### ISSUE TITLE
[Bug] Flip icon feature does not work on radio button item

### ISSUE THREAD

**Issue by pylapp** [Orange-OpenSource/ouds-ios#675]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Pro

### OS Version

iOS 18.4.1

### App version

0.15.0

### App build number

1747948186

### OUDS iOS library version embeded in the app

0.14.0

### Token library version embeded in the app

0.11.0

### App build type

beta

### App orientation

Portrait 

### Device language

French

### App language

French

### Describe the issue

Given the demo application on the radio button item screen
When I toggle the flip icon button
The icons of the radio button does not flip

See https://github.com/Orange-OpenSource/ouds-ios/issues/266#issuecomment-2904078767

### Expected Behavior

The icon must flip 

### Accessibility settings

None

**Comment by B3nz01d** [comment]
Completed during the sprint 19
